src/main/java/reactor/netty/http/HttpDecoderSpec.java

```diff
@@ -131,6 +134,32 @@ public int initialBufferSize() {
 		return initialBufferSize;
 	}
 
+	/**
+	 * Configure the maximum length of the content of the H2C upgrade request.
```

Comment:
Maybe make it more clear since its a builder spec with the full name of h2c: http2ClearText

Reply:
fixed